pkg/api/check_html_output.go

```diff
@@ -0,0 +1,45 @@
+package api
+
+import (
+	"encoding/json"
+	"net/http"
+)
+
+type checkEntry struct {
+	Actor   string   `json:"check_actor"`
+	Action  string   `json:"check_action"`
+	Status  string   `json:"status"`
+	Summary string   `json:"summary"`
+	Params  []string `json:"params"`
+}
+
+type checks struct {
+	Checks []checkEntry `json:"checks"`
+}
+
+type checkHTMLOutputParams struct {
+	CheckOutput []checks `json:"check_output"`
```

Comment:
In hindsight, it'd be better to use `values` here [1], so we could use `ObjValue` in this struct. 

I think it's OK to change this in another opportunity, though.

[1] https://github.com/leapp-to/leapp-actors/pull/19/files#diff-6b3a6f81f69e8399e64fe61b9c2e5186R17